packages/discovery-provider/src/queries/get_comments.py

```diff
@@ -397,6 +397,141 @@ def filter_mentions(mention):
         return track_comment_res
 
 
+def get_user_comments(args, user_id, current_user_id):
```

Comment:
Do you mind using a typed dictionary for input args here and defining a response type?
[Example of TypedDict](https://github.com/AudiusProject/audius-protocol/blob/957a0b72257b3f20b35493accda02830d84af289/packages/discovery-provider/src/queries/get_audio_transactions_history.py#L22)
[Example of typed response](https://github.com/AudiusProject/audius-protocol/blob/957a0b72257b3f20b35493accda02830d84af289/packages/discovery-provider/src/queries/get_associated_user_wallet.py#L14)

Then calling it looks something like `get_user_comments(GetUserCommentsArgs({ id=..., user_id=... })`